app/views/tasks/index.html.haml

```diff
@@ -13,7 +16,7 @@
     - @tasks.each do |task|
       %tr
         %td=task.name
-        %td=task.completed
+        %td{class: "complete"}=task.completed
```

Comment:
Don't see this being used in any stylesheets, can you think of a different way to assert the count of completed tasks in the feature spec without having to add a class for the spec?

Reply:
Got rid of and assert the second td in tr.